library/common/jni/jni_interface.cc

```diff
@@ -1,19 +1,18 @@
-#include <android/log.h>
-#include <ares.h>
 #include <jni.h>
 
 #include <string>
 
+#include "library/common/jni/jni_utility.h"
 #include "library/common/main_interface.h"
 
-static JavaVM* static_jvm = nullptr;
+JavaVM* global_jvm = nullptr;
```

Comment:
I have some misgivings about making this global. Even a static is a little bit iffy. Can we at least wrap it in a read-only accessor?

Reply:
The fact that everyone should be calling `get_env()` is all the more reason to me to hide, rather than expose it.